draft-js-emoji-plugin/src/index.js

```diff
@@ -198,8 +200,12 @@ export default (config = {}) => {
     handleReturn: (keyboardEvent) => callbacks.handleReturn && callbacks.handleReturn(keyboardEvent),
     onChange: (editorState) => {
       let newEditorState = attachImmutableEntitiesToEmojis(editorState);
+      const isEditorEditable = !getEditorReadOnly();
+      const isContentDifferent = !newEditorState.getCurrentContent().equals(
+        editorState.getCurrentContent()
+      );
 
```

Comment:
this looks great.

Could you please:
- use getReadOnly from the `onChange` handler (`onChange` also gets passed an object that contains all the plugin methods like `getEditorState` or `setEditorState` or `getReadOnly`)
- also if you could please bump the version (a minor patch) and add an item to the change log :heart: